docs/src/pages/components/data-grid/editing/BasicEditingGrid.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import { DataGrid, GridColumns, GridRowsProp } from '@material-ui/data-grid';
+import {
+  randomCreatedDate,
+  randomTraderName,
+  randomUpdatedDate,
+} from '@material-ui/x-grid-data-generator';
+
+const columns: GridColumns = [
+  { field: 'name', headerName: 'Name', width: 180, editable: true },
+  { field: 'age', headerName: 'Age', type: 'number', editable: true },
+  {
+    field: 'dateCreated',
+    headerName: 'Date Created',
+    type: 'date',
+    width: 180,
+    editable: true,
+  },
+  {
+    field: 'lastLogin',
+    headerName: 'Last Login',
+    type: 'dateTime',
+    width: 220,
+    editable: true,
+  },
+];
+
+const rows: GridRowsProp = [
+  {
+    id: 1,
+    name: randomTraderName(),
+    age: 25,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 2,
+    name: randomTraderName(),
+    age: 36,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 3,
+    name: randomTraderName(),
+    age: 19,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 4,
+    name: randomTraderName(),
+    age: 28,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+  {
+    id: 5,
+    name: randomTraderName(),
+    age: 23,
+    dateCreated: randomCreatedDate(),
+    lastLogin: randomUpdatedDate(),
+  },
+];
+
+export default function BasicEditingGrid() {
+  return <DataGrid rows={rows} columns={columns} autoHeight />;
```

Comment:
This format for the demos is not possible. We can't use `autoHeight` without setting the dimensions of the demo. See the following for why:

Initial render: 

<img width="652" alt="Screenshot 2021-04-08 at 11 38 14" src="https://user-images.githubusercontent.com/3165635/114004466-f9c95080-985e-11eb-84ed-65a4f0050e50.png">

---

Hydration:

<img width="647" alt="Screenshot 2021-04-08 at 11 38 24" src="https://user-images.githubusercontent.com/3165635/114005081-82e08780-985f-11eb-8736-b2c54a399c47.png">


The height of the page changes, this is messing the #link which creates a low-quality DX when browsing the docs.

x the other demos